doc/asyncomplete.txt

```diff
@@ -5,9 +5,9 @@
 ===============================================================================
 CONTENTS                                                *asyncomplete-contents*
 
-    1. Introduction        |asyncomplete-introduction|
-    2. Options             |asyncomplete-options|
-
+    1. Introduction              |asyncomplete-introduction|
```

Comment:
should use spaces instead of tabs.

Reply:
Ups, sorry, I'll fix it! Looks like some plugin kindly inserted tabs for me, I personally don't use them...

Reply:
Hm, I've checked the doc and haven't found any tabs there. Are you sure they are there? I use `expandtab` in my vimrc, so I think even if I press `tab` it inserts four spaces automatically. I tried to highlight them with `set list && set listchars=tab:>-` but it showed me nothing (I checked that this command works for me if I force vim to insert tab via ctrl-v + tab).

Reply:
I will fix this once merged.